src/EditorFeatures/CSharpTest/QuickInfo/SyntacticQuickInfoSourceTests.cs

```diff
@@ -281,21 +280,25 @@ private QuickInfoProvider CreateProvider(TestWorkspace workspace)
         protected override async Task AssertContentIsAsync(
             TestWorkspace workspace,
             Document document,
+            ITextSnapshot snapshot,
             int position,
             string expectedContent,
             string expectedDocumentationComment = null)
         {
             var provider = CreateProvider(workspace);
             var info = await provider.GetQuickInfoAsync(new QuickInfoContext(document, position, CancellationToken.None));
             Assert.NotNull(info);
-
             Assert.NotEqual(0, info.RelatedSpans.Length);
-            var tabSize = document.Project.Solution.Workspace.Options.GetOption(Microsoft.CodeAnalysis.Formatting.FormattingOptions.TabSize, document.Project.Language);
-            var text = await document.GetTextAsync();
 
-            var classifiedSpans = info.RelatedSpans.Select(s => new ClassifiedSpan(ClassificationTypeNames.Text, s));
-            var spans = IndentationHelper.GetSpansWithAlignedIndentation(text, classifiedSpans.ToImmutableArray(), tabSize);
-            var actualText = string.Concat(spans.Select(s => text.GetSubText(s.TextSpan).ToString()));
+            var trackingSpan = new Mock<ITrackingSpan>();
+            var quickInfoItem = await IntellisenseQuickInfoBuilder.BuildItemAsync(trackingSpan.Object, info, snapshot, document, CancellationToken.None);
+            var containerElement = quickInfoItem.Item as ContainerElement;
+
+            var textElements = containerElement.Elements.Where(e => e is ClassifiedTextElement);
```

Comment:
There's .OfType<ClassifiedTextElement>() which also means you can avoid the cast later.